components/daikin_s21/climate/daikin_s21_climate.cpp

```diff
@@ -381,7 +442,9 @@ void DaikinS21Climate::set_s21_climate() {
   // in Home Assistant as stale state is published over the commanded state
   // followed by the active state.
   this->command_active = true;
-  this->set_timeout(command_timeout_name, DaikinS21Climate::state_publication_timeout_ms, std::bind(&DaikinS21Climate::command_timeout_handler, this));
+  this->set_timeout(command_timeout_name,
+                    DaikinS21Climate::state_publication_timeout_ms + this->get_parent()->get_update_interval(), // extend the timeour if S21 is polling to wait for slower updates
```

Comment:
Dynamic timeout when polling, discussed when adding periodic readout support to S21